index.html

```diff
@@ -5,8 +5,76 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link rel='stylesheet' href='styles/index.css'/>
 </head>
+
+
 <body>
+
     
+
+    <section class='wholeThing'>
+        <section>
+            
+        </section>
+        <header class="app_name">
+            <h1>"WEATHER REPORT"</h1>
+        </header>
+
+        
+
+        <section class='city-name_section'>
+            <h2>Where would you like to go?</h2>
+            <input type='text' id='cityNameInput' value='Los Angeles'>
+            <button id="cityNameReset" class="city-name__reset-btn">Reset</button>
+        </section>
+
+        <section>
+            <div id='skyLeft' class='sky_sunny'>
+                <div id='horizon'>☁️ ☁️ ☁️ ☀️ ☁️ ☁️</div>
+        </section>
+
+
+        <section class='temperature_section'>
+            <h2>Today's Temp</h2>
+            <div class='temperature_content'>
+                <span id='increaseTempControl'>⬆️</span>
+                <span id='tempValue' class='red-orange'>83</span>
+                <span id="decreaseTempControl">⬇️</span>
+            </div>
+        </section>
+
+        <section>
+            <div id='skyRight' class='sky_sunny'>
+                <div id='horizon'>☁️ ☁️ ☁️ ☀️ ☁️ ☁️</div>
+        </section>
+        <section>
+            <div id='gardenLeft' class='garden_sunny'>
+                <div id='landscape'>🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂</div>
+            </div>
+        </section>
+
+        <section class='sky_section'>
+            Welcome To
```

Comment:
normally you would want to put some type of tag around the text. You can use a <p> here